### PR TITLE
Security weaponry and balance tweaks.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Let's go over each of the problems that this PR attempts to fix.

1. Uplink powercreep.
With the amount of weapons and tools that antags now have in their uplink, security is at a severe disadvantage -- sanasomnum, combat stims, slugs and assault shotguns, marksman rifles. The security armoury is no longer enough, mainly because one officer will never be armed with lethals, due to there only being two laser rifles and one burst rifle. As a result, one burst rifle has been added to the armoury.
With this in mind, heavy armour and ballistic armour have both had their ballistic resist upped by a bit to compensate. This should mean that fights should, again, last slightly longer. Polymer ammunition was updated to do around the same amount of damage as before.

2. Species with high brute resist.
To put it bluntly, G2s with heavy armour and LMGs have no counter in game, if they cannot be flashbanged. The PEAC does not work as intended since it has no AP, and even with the AP it wouldn't be a very scary weapon due to G2 brute resist. It now does 4x the damage to synthetic targets instead of 3x and has higher AP, which will allow it to blow through heavy armour (you're looking at about 120dmg to G2s).

3. Full auto.
Full auto is generally problematic due to being overwhelmingly strong. It allows you to put 5 bullets into someone per second (it has a 0.1s firing delay!). This fixes full auto being extremely oppressive and straight up better than both burst and semiauto by adding 0.5s delay to full auto globally.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by KingOfThePing — APPROVED** [review]

File: code/modules/projectiles/gun.dm [hunk 1]
```diff
@@ -520,7 +520,7 @@
 	if(length(firemodes))
 		F = firemodes[sel_mode]
 	if(one_hand_fa_penalty > 2 && !wielded && F?.name == "full auto") // todo: make firemode names defines
-		P.accuracy -= one_hand_fa_penalty/2
+		P.accuracy -= one_hand_fa_penalty / 2
```
Comment (Arrow768) [inline code/modules/projectiles/gun.dm:523]:
```suggestion
		P.accuracy -= one_hand_fa_penalty * 0.5
```

**Review by SleepyGemmy — APPROVED** [review]

**Comment by Butterrobber202** [comment]
> With the amount of weapons and tools that antags now have in their uplink, security is at a severe disadvantage -- sanasomnum, combat stims, slugs and assault shotguns, marksman rifles. The security armoury is no longer enough, mainly because one officer will never be armed with lethals, due to there only being two laser rifles and one burst rifle. As a result, one burst rifle has been added to the armoury.

On this, Security will almost always have a numbers advantage on antags, be they mercs, traitors, raiders or otherwise. If their gear is 1:1 with antagonist uplink stuff, antagonists will have to cheese to break down sec numbers. And even if Sec _does_ fall, the crew has the crew armory to fall back on.

**Comment by Tag114** [comment]
Rapid-clicking with fulll auto lets you fire faster than holding the mouse button down.

**Pull request merged**